### PR TITLE
Update to add the Docker build files

### PR DESCRIPTION
**Pull request by rsgrewal-aws**
Description of changes: Added the files needed to review the docker build

Testing done: Completed it runs end to end

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [ x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [ x] I have tested my notebook(s) and ensured it runs end-to-end
- [ x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb [hunk 1]
```diff
@@ -0,0 +1,550 @@
+{
```
Comment (jkroll-aws) [inline advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>Branding nit: <code>Catboost</code> -&gt; <code>CatBoost</code>	(2-3 occurrences)</p><p> </p><p>Typo: <code>demostrate</code> -&gt; <code>demonstrate </code>(2 occurrences)</p><p> </p><p>Grammar: <code>show case</code> -&gt; <code>showcase</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jkroll-aws) [inline advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>Grammar nit: <code>This include</code> -&gt; <code>This includes</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (jkroll-aws) [inline advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>Typo: <code>moels</code> -&gt; <code>models</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>Thank you for catching this , have fixed in the latest push</p>

Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>Fixed in the latest push</p>

Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb:1]: <p>fixed in the latest push</p>

File: advanced_functionality/multi_model_catboost/container/dockerd-entrypoint.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import subprocess
```
[the remark this answers is not in the document]
Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/container/dockerd-entrypoint.py:1]: just updated in the latest commit

File: advanced_functionality/multi_model_catboost/container/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    apt-get -y install --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    openjdk-8-jdk-headless \
+    python3-dev \
+    curl \
+    python3 \
+    vim \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -O https://bootstrap.pypa.io/pip/3.6/get-pip.py \
```
[the remark this answers is not in the document]
Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/container/Dockerfile]: changed to 3.7

File: advanced_functionality/multi_model_catboost/container/dockerd-entrypoint.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/container/dockerd-entrypoint.py:1]: ran the latest black file

File: advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb (hunk 1 above)
Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb]: fixed

Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb]: fixed

Comment (rsgrewal-aws) [reply advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb]: fixed

**Comment by rsgrewal-aws** [comment]
made the required changes

**Review by rsgrewal-aws — COMMENTED** [review]
made all the changes

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**